eventhub_endpoint.bal

```diff
@@ -239,16 +234,17 @@ public client class Client {
     #
     # + eventHubPath - event hub path
     # + return - Return revoke publisher or Error
-    public remote function getRevokedPublishers(string eventHubPath) returns @tainted xml|error {
-        http:Request req = self.getAuthorizedRequest();
-        var response = self.clientEndpoint->get("/" + eventHubPath + "/revokedpublishers", req);
+    remote function getRevokedPublishers(string eventHubPath) returns @tainted xml|error {
+        http:Request req = getAuthorizedRequest(self.config);
+        var response = self.clientEndpoint->get("/" + eventHubPath + "/revokedpublishers" + 
```

Comment:
If we can have a function to generate URL ( from a ```string[]`` for example) it will be more cleaner.

Reply:
Using an additional function reduce the readability of the code